tasks/update-code/rsync_direct.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: ANSISTRANO | RSYNC | Get Release path (in rsync_direct case)
+  command: echo "{{ ansistrano_release_path.stdout }}"
+  register: ansistrano_release_rsync_copy_path
```

Comment:
Is this extra variable needed? You can use ansistrano_release_path.stdout in the dest property, right? Not a biggie but for the sake of clean coding